src/DynamoCore/Search/SearchElements/CustomNodeSearchElement.cs

```diff
@@ -49,6 +49,10 @@ public void SyncWithCustomNodeInfo(CustomNodeInfo info)
             FullCategoryName = info.Category;
             Description = info.Description;
             Path = info.Path;
+            if (info.IsPackageMember)
+                ElementType = ElementTypeEnum.Package;
+            else
+                ElementType = ElementTypeEnum.CustomNode;
```

Comment:
After the above change, here this can be updated to:

```csharp
ElementType = ElementTypes.CustomNode;
if (info.IsPackageMember)
    ElementType |= ElementTypes.Packaged; // Add one more flag.
```